src/PluginSession.php

```diff
@@ -260,4 +292,32 @@ public function isUserView() {
 		return $this->userView;
 	}
 
+	/**
+	 * Destroy the session with the given id
+	 *
+	 * @param String $sessionId
+	 * @return bool true on success or false on failure.
+	 */
+	public function destroySession(String $sessionId = null) {
+
+		$sessionId = $sessionId ?: $this->sessionId;
```

Comment:
You could remove this line, if you set the default argument to  `destroySession(String $sessionId = $this->sessionId)`

Reply:
this is new to me but very helpful!